src/comps/Deposit.tsx

```diff
@@ -411,7 +419,14 @@ const DepositFlowConfirm = ({
       );
       // convert buffer to hex
       const depositScriptHexPreHash = uint8ArrayToHexString(depositScript);
+      const p2trAddress = createDepositAddress(
+        serializedAddress,
+        signerPubKey,
+        maxFee,
+        lockTime
+      );
```

Comment:
`createDepositAddress` seems to redo some computation we do above in here related to the scripts. What about having `createDepositAddress` (eventually renaming it) to return an object with all the required info, so that all the logic is in a single place?

Reply:
This has been consolidated